membership_account_balance/README.rst

```diff
@@ -0,0 +1,27 @@
+Membership Management - Member Account Balance
```

Comment:
To be proposed to the OCA directly

Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst